crates/ra_mbe/src/lib.rs

```diff
@@ -337,4 +337,45 @@ SOURCE_FILE@[0; 40)
         );
     }
 
+    #[test]
+    fn expand_literals_to_item_list() {
+        fn to_subtree(tt: &tt::TokenTree) -> &tt::Subtree {
+            if let tt::TokenTree::Subtree(subtree) = tt {
+                return &subtree;
+            }
+            assert!(false, "It is not a subtree");
+            unreachable!();
```

Comment:
That can be shortened to `unreachable!("It is not a subtree")`

Reply:
Fixed